### ISSUE TITLE
Internet Explorer

### ISSUE THREAD

**Issue by adamkleingit** [500tech/angular2-tree-component#5]
Test on IE

**Comment by loliparra80** [comment]
Hello Adam, I'm testing your example (systemjs folder) but when I try on IExplore the node expand event doesn't working. When I push on the arrow of a node, the node doesn't expand.

How can I fix it?

By the way I don't know exactly how to chante the styles to put different images on nodes and children, can you give an example?

Thank you.

**Comment by adamkleingit** [comment]
Hi,
I'm aware that there are IE problems, I will try to fix them ASAP. Does it work for you on Chrome / Firefox?

You should use a custom template.
Inside the custom template you can check node.level, or node.isRoot, or node.isLeaf, or access all of you node's data on node.data.
Based on whatever you like you can set different images.
e.g.:
```
'
  <img *ngIf="node.isLeaf" src="leaf-node">
  <img *ngIf="!node.isLeaf" src="parent-node">
'
```

**Comment by jFensch** [comment]
These lines will not work in IE/Edge without polyfill:
https://github.com/500tech/angular2-tree-component/blob/222f7c450c91d963d97d2ce9b0854e4f58f0d95b/lib/components/tree.component.ts#L91-96

Currently finding success with this in our polyfills: https://www.npmjs.com/package/element-closest

**Comment by adamkleingit** [comment]
Solved in 1.1.9
